static/quickstart.js

```diff
@@ -77,30 +45,20 @@
     })
     .catch(function (err) {
       console.log(err);
-      log("Could not get a token from server!");
+      log("Error starting client!");
     });
     
   // Bind button to make call
   document.getElementById('button-call').onclick = function () {
     // get the phone number to connect the call to
     var params = {
-      To: document.getElementById('phone-number').value
+      phone: document.getElementById('phone-number').value
     };
 
-    console.log('Calling ' + params.To + '...');
+    console.log('Calling ' + params.phone + '...');
     if (device) {
-      var outgoingConnection = device.connect(params);
-      outgoingConnection.on("ringing", function () {
-        log("Ringing...");
-      });
-    }
-  };
-
-  // Bind button to hangup call
-  document.getElementById('button-hangup').onclick = function () {
-    log('Hanging up...');
-    if (device) {
-      device.disconnectAll();
+      var outgoingCall = device.connect({params: params});
+      outgoingCall.then(callEstablished);
```

Comment:
Two teeny nits:

- We can use object shorthand here to reduce the noise of `params = params`
- If `outgoingCall` isn't being referenced again later, and `connect` returns a promise, might be simpler to just chain immediately and not store `outgoingCall`
```suggestion
      device.connect({ params }).then(callEstablished);
```